File: ws/verdugo/u2/ws32ProgramInC#/ChickenFarmSimulator/ec.edu.espe.chickenfarm.model/ChickenFarmer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ChickenFarmSimulator.ec.edu.espe.chickenfarm.model
{
    internal class ChickenFarmer
    {
        public string name { get; set; }
        public ChickenCoop[] coops { get; set; }


        public void add(ChickenCoop coop)
        {

        }

        public void remove(int coopId)
        {

        }

        public void resetlteration()
        {

        }

        public ChickenCoop next()
        {
            return new ChickenCoop();
        }
    }
}

File: ws/rivadeneira/u2/ws32 QuizzVisualBasic/ConsoleApp1/ConsoleApp1/ec.edu.espe.chicken.model/Chicken Farmer.vb
Public Class Chicken_Farmer
    Private name As String
    Private coops() As Coop

    Public Sub New(name As String, coops() As Coop)
        Me.name = name
        Me.coops = coops
    End Sub

    Public Property Name1 As String
        Get
            Return name
        End Get
        Set(value As String)
            name = value
        End Set
    End Property

    Public Property Coops1 As Coop()
        Get
            Return coops
        End Get
        Set(value As Coop())
            coops = value
        End Set
    End Property
End Class
